### PR TITLE
Add Ability to Toggle Offline|Online status

### PR DESCRIPTION
**Pull request by RedSparr0w**
- Start Login in "appear offline" mode
- Once idling option down bottom left to show as online | offline

### CODE REVIEW

File: farmer.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ client.on('steamGuard', function(domain, callback, lastcode) {
 });
 
 client.on('loggedOn', function() {
-	client.setPersona(3);
+	client.setPersona(0);
```
Comment (DoctorMcKay) [inline farmer.js]: Prefer to use the enum values instead of magic numbers here.

File: html/index.html [hunk 2]
```diff
@@ -70,7 +70,14 @@
 		<img src="" alt="Image Not Found" />
 		<h4></h4>
 		<p></p>
-		<div id="RefreshIn"><span id="NewItems" class="label label-pill" data-toggle="tooltip" data-placement="right" title="New Items"></span>Refresh in <div id="RefreshTime"></div></div>
+		<div id="RefreshIn">
+			<span id="NewItems" class="label label-pill" data-toggle="tooltip" data-placement="right" title="New Items"></span>
+			<span id="onlineStatus" class="label label-pill" data-toggle="tooltip" data-placement="top" title="" data-original-title="Online Status">
+				<span id="goOnline" onclick="client.setPersona(1);">Appear Offline</span>
```
Comment (DoctorMcKay) [inline html/index.html]: ... and here

**Comment by RedSparr0w** [comment]
Have updated

Don't merge just yet, need to check it when i get home.

**Comment by RedSparr0w** [comment]
Tested and all working fine [=

**Pull request merged**